io.github.Adda0.Stardrop.yaml

```diff
@@ -0,0 +1,76 @@
+app-id: io.github.Adda0.Stardrop
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet7
+build-options:
+    append-path: /usr/lib/sdk/dotnet7/bin
+    append-ld-library-path: /usr/lib/sdk/dotnet7/lib
+    env:
+        PKG_CONFIG_PATH: /app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig
```

Comment:
please use `prepend-pkg-config-path`.

Reply:
yes the doc for the SDK needs to be updated.

Reply:
Fixed.